demo/src/app/components/progressbar/demos/config/progressbar-config.ts

```diff
@@ -0,0 +1,17 @@
+import {Component} from '@angular/core';
+import {NgbProgressbarConfig} from '@ng-bootstrap/ng-bootstrap';
+
+@Component({
+  selector: 'ngbd-progressbar-config',
+  templateUrl: './progressbar-config.html',
+  providers: [NgbProgressbarConfig] // add the NgbProgressbarConfig to the component providers
```

Comment:
If we don't add the provider here, the config will be the global one, and this demo will thus change the global config used to initialize all the other demos.